src/ray/gcs/redis_context.cc

```diff
@@ -18,13 +21,24 @@ namespace {
 /// A helper function to call the callback and delete it from the callback
 /// manager if necessary.
 void ProcessCallback(int64_t callback_index, const std::string &data) {
-  if (callback_index >= 0) {
-    bool delete_callback =
-        ray::gcs::RedisCallbackManager::instance().get(callback_index)(data);
-    // Delete the callback if necessary.
-    if (delete_callback) {
-      ray::gcs::RedisCallbackManager::instance().remove(callback_index);
-    }
+  RAY_CHECK(callback_index >= 0) << "The callback index must be greater than 0, "
+                                 << "but it actually be " << callback_index;
+  auto callback_item = ray::gcs::RedisCallbackManager::instance().get(callback_index);
+  if (!callback_item.is_subscription) {
+    // Record the redis latency for non-subscription redis operator.
+    auto end_time = current_sys_time_us();
+    RAY_LOG(DEBUG) << "Record a redis latency, start time is " << callback_item.start_time
+                   << "us, end time is " << end_time << "us, and the elapsed time is "
+                   << end_time - callback_item.start_time << "us.";
```

Comment:
Is this debug log useful?

Reply:
This log msg is used to check the latency metric result when I debuged, and it is not used any more. Let me remove it.